vertx-sql-client/src/main/java/io/vertx/sqlclient/spi/DatabaseMetaData.java

```diff
@@ -0,0 +1,26 @@
+package io.vertx.sqlclient.spi;
+
+import io.vertx.codegen.annotations.VertxGen;
+
+/**
+ * Contains static metadata about the backend database server 
+ */
+@VertxGen
+public interface DatabaseMetaData {
```

Comment:
`DatabaseMetadata` , metadata is a single word (https://en.wikipedia.org/wiki/Metadata)

Reply:
This will be a fun review comment to fix on a case since I'm on a case insensitive OS 🙃

Reply:
sometimes I do ahve this too, here is how I deal with:

1/ rename the file to something else like Foo
2/ commit
3/ rename Foo to the old name with case sensitivity change
4/ commit
5/ squash commits